hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -266,6 +266,12 @@ def in_maintenance_mode?
       @admin.isMasterInMaintenanceMode
     end
 
+    #----------------------------------------------------------------------------------------------
+    # Request HBCK chore to run
+    def hbck_chore_run
+      @admin.runHbckChore
```

Comment:
Yeah, good question by @duo regards whether it should be in hbcksevice or not. It probably should?

Reply:
OK.